src/platform/site-wide/sass/modules/_m-footer.scss

```diff
@@ -102,10 +102,13 @@
       > a {
         display: inline-block;
         color: $color-white;
-        text-decoration: none;
         font-size: 1em;
         font-weight: 400;
         margin: .25em 0;
+
+        &:hover {
+          color: $color-gold !important;
+        }
```

Comment:
I'm likely missing it but is this part of the ticket too? The link text color changing to gold on hover?

Reply:
Typically it seems like the underline is what is added on hover (although the gold seems like way more fun)

Reply:
It's what we currently do on Desktop, and it was the suggested css in the ticket  -

![image](https://user-images.githubusercontent.com/14869324/104651552-0af62d00-5675-11eb-9ebb-55be4b23acb2.png)

Reply:
Though I am not sure how useful it is knowing users on tablets and phones don't use a mouse 😆

Reply:
Ah cool! Thanks for the info!